test-zeekoe.py

```diff
@@ -146,6 +147,14 @@ def scenario_close_with_expiry(config, channel_name, verbose):
     # TODO: then customer should detect and respond with cust close
     pass
 
+def check_blockchain_maturity(url):
+    full_url = url + "/chains/main/blocks/head/metadata"
+    r = requests.get(url = full_url)
+    data = r.json()
+    level = data['level']['level']
+    assert(level >= 60), """The blockchain level is less than 60 blocks. 
+    Wait until it is at least 60 blocks before running any tests"""
```

Comment:
Could this be a constant at the top of the file, commented with a summary of the reasoning?

Reply:
done!